src/Avalonia.Visuals/Properties/AssemblyInfo.cs

```diff
@@ -10,5 +10,7 @@
 [assembly: XmlnsDefinition("https://github.com/avaloniaui", "Avalonia.Media")]
 [assembly: XmlnsDefinition("https://github.com/avaloniaui", "Avalonia")]
 
+[assembly: InternalsVisibleTo("Avalonia.Direct2D1")]
```

Comment:
We shouldn't really be doing this - it should be possible for people to write their own rendering backends without us adding an `InternalsVisibleTo` attribute.